Gifski/Gifski.swift

```diff
@@ -4,53 +4,54 @@ import AVFoundation
 final class Gifski {
 	private(set) var isRunning = false
 	private var progress: Progress!
+    private var observation: NSKeyValueObservation?
```

Comment:
Incorrect indentation here

Reply:
Looks like these indentation error is caused by my Xcode being set to 4 spaces instead of tabs. Will set to tabs and fix!